cds-snc/notification-terraform
# Setting staging to k8s 1.28

# Summary | Résumé

Upgrading staging to k8s 1.28

# Test instructions | Instructions pour tester la modification

- Tested in scratch
- Smoke test staging
- Verify admin works in staging
- Verify logging in staging
- Verify all pods in staging are running correctly
